mariadb-database-id/3.initialize-database.sql

```diff
@@ -1,15 +1,16 @@
 USE eventuate;
 
 CREATE TABLE new_message (
```

Comment:
@cer Chris,

I used 3 testing scenarios:

1) old schema - new code that uses application id generation
2) new schema - new code that uses application id generation 
3) new schema - new code that uses database id generation

If you think that first point is not necessary, I will remove it.
However, as I wrote above, having 2 schemes is a way to show supposed migration approach and smoke test it.

Reply:
I suppose it's worthwhile to test against the old schema.
One thing that might simplify things is that rather than having separate images for old and new id generation, you can have a single image that has a env. variable, which specifies the id generation scheme.

Reply:
A similar approach would be used for the payload column type - e.g. varchar, JSON, ... - no need for a separate image.

Reply:
ok